### PR TITLE
Enable nose2 test timer

### PR DESCRIPTION
**Pull request by jantonguirao**
Signed-off-by: Joaquin Anton <janton@nvidia.com>

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [ ] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [ ] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by jantonguirao** [comment]
!build

**Review by JanuszL — APPROVED** [review]

**Comment by jantonguirao** [comment]
!build

**Comment by jantonguirao** [comment]
!build

File: qa/test_template.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ topdir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )/..
 source $topdir/qa/setup_test_common.sh
 
 # Set runner for python tests
-python_test_runner_package="nose nose2"
+python_test_runner_package="nose nose2 nose2-test-timer"
```
Comment (JanuszL) [inline qa/test_template.sh:12]:
```suggestion
python_test_runner_package="nose nose2 nose-timer nose2-test-timer"
```

File: qa/test_template.sh [hunk 2]
```diff
@@ -18,7 +18,7 @@ python_invoke_test="${python_test_runner} ${python_test_args}"
 # During the transition we run both
 # When all tests are ported old will be removed
 python_new_test_runner="python -m nose2"
```
Comment (JanuszL) [inline qa/test_template.sh:20]: -`-with-timer --timer-color --timer-top-n 20`

**Comment by jantonguirao** [comment]
!build

**Comment by jantonguirao** [comment]
!build

**Review by szalpal — APPROVED** [review]

**Review by JanuszL — APPROVED** [review]

**Pull request merged**